frontend/src/scenes/saved-insights/SavedInsights.tsx

```diff
@@ -299,31 +280,9 @@ export function NewInsightButton({ dataAttr }: NewInsightButtonProps): JSX.Eleme
                     placement: 'bottom-end',
                     className: 'new-insight-overlay',
```

Comment:
Wow, this dropdown is really  w i d e! I haven't notice before, because, well, I've always used the plus in the sidebar. But to ship this, let's constraint the width of `.new-insight-overlay` to something like 21rem (this number seems to result in all options having roughly equal-size two lines of description)